src/lib/DebouncingBoolean.java

```diff
@@ -0,0 +1,50 @@
+package lib;
+import java.util.LinkedList;
+public class DebouncingBoolean implements DebouncedBoolean{
+	private static LinkedList<Boolean> queue;
+	private static int windowSize;
+	private boolean count;
```

Comment:
This should be an `int` keeping track of how many true or false values have been added